app.py

```diff
@@ -20,5 +24,16 @@ async def add_role(member: discord.Member, role_id: int):
     role: discord.Role = member.guild.get_role(config.ROLE_ON_JOIN_ID)
     await member.add_roles(role)
 
+@bot.event
+async def on_ready():
+    def message_cog_maker(prompt):
+        if(prompt['type']=='question'):
+            bot.add_cog(ScheduledMessage(bot, prompt['message'], prompt['channelID'], prompt['intervalSeconds']))
+    if (os.path.isfile('./messages.json')):
+        f = open('./messages.json', 'r')
+        messages = json.loads(f.read())
+        list(map(message_cog_maker, messages['prompts']))
```

Comment:
Neat! However, I think it would be more readable to use a simple for-loop here:
```python
for prompt in messages["prompts"]:
  message_cog_maker(prompt)
```
Also we don't need to turn the result into a list, since we are just iterating over the list of prompts.